tests/js/jest.config.js

```diff
@@ -34,6 +38,7 @@ module.exports = {
 		'<rootDir>/node_modules',
 		'<rootDir>/build',
 	],
+	modulePathIgnorePatterns: [ '<rootDir>/.vscode' ],
```

Comment:
When using the GitHub Copilot plugin, it updates local VSCode settings with regularity when switching back to VSCode, causing watched tests to re-run without the source files changing. Adding this path to `modulePathIgnorePatterns` prevents this from happening.